README.md

```diff
@@ -376,7 +376,7 @@ After that, run the `make-sync-function` script on your document definitions to
 
 Next, create a new spec file in your project's `test/` directory (e.g. `test/foobar-spec.js`) and import the test helper module into the empty spec:
 
-    const testHelper = require('../node_modules/synctos/etc/test-helper.js');
+    var testHelper = require('../node_modules/synctos/etc/test-helper.js');
```

Comment:
Since `const` is an ECMAScript 6 feature that is not supported in all environments, use `var` for the example instead.